driver-core/src/main/java/com/datastax/driver/core/policies/RollingCount.java

```diff
@@ -0,0 +1,121 @@
+/*
+ *      Copyright (C) 2012-2015 DataStax Inc.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+package com.datastax.driver.core.policies;
+
+import io.netty.util.internal.chmv8.LongAdderV8;
```

Comment:
I'm not comfortable importing a class from a Netty internal package. My immediate concern was OSGi, but after a quick test, it turns out that this package is exported so it works. But still, "internal" means "internal". If we really need it, then at least we should create our own copy of the original `java.util.concurrent.atomic.LongAdder`.